### PR TITLE
CHG-228 Nextech Payments API - new end point to Create Payment Profile By merchantCustomerId

### PR DESCRIPTION
**Pull request by dharmendra-nextech**
Story link:
https://nextech.atlassian.net/browse/CHG-228

### CODE REVIEW

File: index.html
Comment (mukeshnextech) [inline index.html:1]: Looking good

**Review by mukeshnextech — COMMENTED** [review]
Looking Good

File: index.html [hunk 1]
```diff
@@ -1094,6 +1100,39 @@ <h2 id="create-payment-profile-dto">CreatePaymentProfileDto</h2>
...
+                <thead>
+                    <tr>
+                        <th>Name</th>
+                        <th>Description</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>PatientGuid</td>
```
Comment (jonhageman) [inline index.html:1114]: We should refer to this as "MerchantCustomerId" to be consistent.

File: index.html [hunk 2]
```diff
@@ -1094,6 +1100,39 @@ <h2 id="create-payment-profile-dto">CreatePaymentProfileDto</h2>
...
+                        <td>Patients Guid</td>
```
Comment (jonhageman) [inline index.html:1115]: replace with this: "This is the Guid provided by the Nextech merchant. Sometimes referred to as 'Patient Guid'"

File: index.html [hunk 3]
```diff
@@ -1094,6 +1100,39 @@ <h2 id="create-payment-profile-dto">CreatePaymentProfileDto</h2>
...
+                        <td>String</td>
+                    </tr>
+                    <tr>
+                        <td>ProfileName</td>
+                        <td>Name for payment profile</td>
+                        <td>String</td>
+                    </tr>
+                    <tr>
+                        <td>ProcessorPaymentMethodId</td>
+                        <td>Id from processor linked to payment method</td>
```
Comment (jonhageman) [inline index.html:1125]: replace with "Id from processor (Stripe) linked to payment method"

File: index.html [hunk 4]
```diff
@@ -1619,9 +1658,65 @@ <h4 id="example-get-all-patient-types">Example: Refund Payment Response</h4>
...
+                 <thead>
+                     <tr>
+                         <th>Name</th>
+                         <th>Description</th>
+                         <th>Type</th>
+                     </tr>
+                 </thead>
+                 <tbody>
+                     <tr>
+                         <td>patientGuid</td>
```
Comment (jonhageman) [inline index.html:1678]: same as above

File: index.html [hunk 5]
```diff
@@ -1619,9 +1658,65 @@ <h4 id="example-get-all-patient-types">Example: Refund Payment Response</h4>
...
 
             <!-- Payment Profiles Payments -->
-            <!-- Get Payment Profiles -->
-            <h1 id='saving-payments'>Payment Profiles Payments</h1>
+             <h1 id='saving-payments'>Payment Profiles Payments</h1>
+
+             <!-- Get Payment Profiles By Merchant Customer Id-->
+             <h2 id='get-payment-profiles-by-merchant-customer-id'>Get Payment Profiles By Merchant Customer ID</h2>
+             <h4 id="example-get-all-patient-types">Endpoint</h4>
+             <p><code>GET https://payfac.api.nextech.com/api/public/paymentProfiles/byMerchantCustomerId/{patientGuid}</code></p>
```
Comment (jonhageman) [inline index.html:1666]: this should be {merchantCustomerId}

**Review by jonhageman — CHANGES_REQUESTED** [review]

**Review by jonhageman — APPROVED** [review]

**Pull request merged**